tests/parm/MOM_input_hafs

```diff
@@ -972,5 +997,3 @@ USE_NET_FW_ADJUSTMENT_SIGN_BUG = False !   [Boolean] default = True
 ! === module MOM_restart ===
 
 ! === module MOM_file_parser ===
-
-USE_HUYNH_STENCIL_BUG = True
```

Comment:
this parameter needs to be here (false is its default), other wise it will break mpi test (answer change with different PE #) on wcoss2. There is a FMA bug in the code but GFDL haven't figured it out yet.

Reply:
@jiandewang The file "MOM_input_hafs" has been updated. Thanks!